cluster/cluster.py

```diff
@@ -121,17 +95,27 @@ def cluster() -> None:
     conn = get_conn()
     cur = conn.cursor()
 
-    LOGGER.info("Creating pg_repack extension in DB")
-    try:
-        cur.execute("""CREATE EXTENSION pg_repack""")
-        LOGGER.info("pg_repack extension created successfully")
-    except errors.lookup(errorcodes.DUPLICATE_OBJECT):
-        LOGGER.info("pg_repack extension already exists")
-    conn.commit()
-
-    cluster_table(conn, "system_platform", ["rh_account_id"])
-
-    cluster_table(conn, "cve_metadata", ["impact_id"])
+    LOGGER.info("Checking pg_repack extension in DB")
+    cur.execute("""SELECT default_version, installed_version FROM pg_available_extensions WHERE name = 'pg_repack'""")
+    row = cur.fetchone()
+    if row:
+        default_version, installed_version = row
+        if installed_version != default_version:
```

Comment:
why default_version and installed_version must be same? what version would be installed after DROP and CREATE?